recipes/graspy/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "graspy" %}
+{% set version = "0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/neurodata/graspy/archive/v{{ version }}.tar.gz
+  sha256: 00cbd96c19113355c181bd8efa07dd487bbac1ebd9a8bf14d0da1945091a51fb
+
+build:
+  number: 0
+  skip: True  # [py<35]
```

Comment:
Can this be `noarch: python` ? 

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
If I include` noarch: python`, the linter gives me:

"noarch packages can't have selectors. If the selectors are necessary, please remove noarch: python"